### ISSUE TITLE
Jumping to wrong item when scrolling up

### ISSUE THREAD

**Issue by DieGlueckswurst** [Cilestal/clickable_list_wheel_view#1]
First of all, awesome work! 

I implemented ClickableListWheel but there is a behavior which I think is not supposed to be like this. It feels unintuitive. 

When scrolling up and tapping the lower item it scrolls up again. I think it makes more sense if you see it, here is a screenvideo:

https://user-images.githubusercontent.com/40027124/109552027-9b77b800-7ad1-11eb-996f-23ccfe93b5ec.mov

In the video I scroll and then tap on "September" but as you can see it scrolls right back to "August" this only happens if you scroll up. The other way around it is working perfectly fine.

**Comment by ghost** [comment]
The `widget.scrollController.offset` is slightly different when scrolling up or down with a `FixedExtentScrollController`.

In my specific case, adding 1 to `widget.scrollController.offset` when computing `currentIndex` fixed the problem.

```
  int _getClickedIndex() {
    final currentIndex =
        (widget.scrollController.offset + 1) ~/ widget.itemHeight;
    final clickOffset = _getClickedOffset();
    final indexOffset = (clickOffset / widget.itemHeight).round();
    final newIndex = currentIndex + indexOffset;

    if (newIndex < 0 || newIndex >= widget.itemCount) {
      return -1;
    }

    return newIndex;
  }
```

However, I suspect the problem is bigger and the current implementation of the `clickOffset` based on the `MeasureSize` is not correct for all cases.

**Comment by DieGlueckswurst** [comment]
this is not working for me :( But maybe I have an idea: Since this only happens when scrolling, more specifically when scrolling  down and selecting an item from below the selected item:

Maybe you could add a listener variable "isScrolling" and also a "isScrollingUp"/"isScrollingDown" and `if (isScrollingDown) { index = currentIndex - indexOffset } else { index = currentindex + offset }

Does that make sense? I am new to flutter so I am not too sure if this is working but in my head it is :D so maybe you could give that a try?

**Comment by DieGlueckswurst** [comment]
@ThierryDD any updates on this? I tried it by myself but couldn't make it work. The problem is like you said bigger ...

**Comment by Cilestal** [comment]
hm..

**Comment by Cilestal** [comment]
Could you provide the complete code?

**Comment by DieGlueckswurst** [comment]
@Cilestal what do you mean? My code is your code :D I couldn't figure it out yet...

**Comment by DieGlueckswurst** [comment]
@Cilestal  Hi man, do you have any updates on this? I would love to use your package but that bug when scrolling is really not the expected behavior :( I've tried a couple of things myself but couldn't figure it out.

**Comment by Cilestal** [comment]
Fixed in 0.1.0
